packages/dependency-extraction-webpack-plugin/README.md

```diff
@@ -116,6 +116,13 @@ module.exports = {
 
 The output format for the generated asset file. There are two options available: 'php' or 'json'.
 
+##### `outputFilename`
```

Comment:
With this new option proposed `combinedOutputFile` plays a nearly identical role. At some point, we might want to deprecate `combinedOutputFile` and consolidate the handling in `outputFilename`.